src/content/peerconnection/trickle-ice/js/main.js

```diff
@@ -240,6 +238,16 @@ function iceCallback(event) {
       return;
     }
     const {candidate} = event;
+    let candidateStats;
+    if (['srflx', 'relay'].includes(candidate.type)) {
+      const stats = await pc.getStats();
+      stats.forEach(report => {
+        if (!candidateStats && report.type === 'local-candidate' && report.address === candidate.address && report.port === candidate.port) {
+          candidateStats = report;
```

Comment:
should there be a 'break' here, or is there some fear that we'll want to match the last one not the first one?

Reply:
you can't break a .forEach :-(